src/migration/1700735236452-YouTubePost.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class YouTubePost1700735236452 implements MigrationInterface {
```

Comment:
Please merge both migrations if possible. Ideally we should strive to have 1 migration per PR to easily review database changes

Reply:
I don't think the other settings are added via migrations. But I don't mind either way.

If I must add the new setting in this migration, merging will have to wait till the frontend part is completed so we don't show the youtube setting in the wrong place.

Reply:
Ido means the two migration files, not the actual fields.

Although, we should probably open a discussion around that soon 👍

Reply:
Ahh, I misunderstood.

I could merge them both, no objections to it.

My personal preference would be keep them separate as they get introduced in two different commits, and I (strive to) try to keep my commits atomic and each commit can be checked out and the application still works fine.

Reply:
Since we use squash and merge, I prefer the final commit to be easier to understand than individual commits on the side branch. Although I love your approach

Reply:
I never really look at the squashed commit, I usually go directly to see the PR it originated from. Albeit its a bad practice/habit, as it depends on GitHub (or the git host you use) to retain the commits/branch after merging.

Just wish that there was a new option to merge branches. A mix of squash and merge, everything is contained in one squashed commit, but you can "zoom in" and see the commits on the commit.

Reply:
Will rebase and merge the two migrations tomorrow